src/Microsoft.IdentityModel.Abstractions/TelemetryClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Interface for Telemetry tracking.
+    /// </summary>
+    public abstract class TelemetryClient
```

Comment:
Looks like everything except the ClientId is abstract. Should this be an interface instead?

Reply:
Generally agree (see first iteration :)) will close with team this week

Reply:
Changed back to interface